### ISSUE TITLE
AC tool "Organization Merges" broken

### ISSUE THREAD

**Issue by arnoldnipper** [peeringdb/peeringdb#1022]
**Describe the bug**
I've [merged](https://peeringdb.com/cp/peeringdb_server/organizationmerge/1375/change/) iWeb Inc. with iWeb Technologies Inc. However, the [facility iWeb Montreal](https://www.peeringdb.com/fac/1404) still belongs to iWeb Inc.

**To Reproduce**
1. Go to the URLs mentioned above

**Expected behavior**
All objects which belonged to the merged org should now belong to the new org

**Who is affected by the problem?**
All users, AC

**What is the impact?**
Manual work for AC

**Are there security concerns?**
no

**Are there privacy concerns?**
no

**What are the proposed actions?**
fix the bug

**What is the proposed priority?**
normal

**Comment by arnoldnipper** [comment]
@peeringdb/ac, have you seen this behaviour recently?

**Comment by peterhelmenstine** [comment]
I am currently seeing the facility under iWeb Technologies Inc. Assuming you've done a work around in the interim?

**Comment by arnoldnipper** [comment]
> I am currently seeing the facility under iWeb Technologies Inc. Assuming you've done a work around in the interim?

Right. I went to the [AC fac GUI](https://peeringdb.com/cp/peeringdb_server/facility/1404/change/) and changed the Org there.

**Comment by peterhelmenstine** [comment]
I see. I haven't used the tool yet so can't comment on it's functionality but if it merged to the old one that is a problem so +1

**Comment by arnoldnipper** [comment]
See recent [merge](https://peeringdb.com/cp/peeringdb_server/organizationmerge/1379/change/). My guestimate is that this happens when a dependant object is edited at the same time. I edited the [network](https://peeringdb.com/net/27764).

@peeringdb/20c and @vegu, could you please have a look as I left the net hanging off the deleted org.

**Comment by arnoldnipper** [comment]
@vegu writes: Due to the static nature of the network page you have open it will not be aware of the changes made by the merge tool until you reload the page. This seems more of an unintended use issue rather than a bug. If this workflow is something that would be beneficial to support we think that would classify as a feature enhancement.

**Comment by peterhelmenstine** [comment]
Feature enhancement makes sense, not a bug now that we know how it works and is intended.

**Comment by arnoldnipper** [comment]
As this race-condition also may happen when AC and the user are working simultaneously on the (related) objects it makes sense to have this as a feature.

Iow: if AC makes a merge they can 100% trust that all objects are handled properly.
